src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -3769,9 +3769,11 @@ def spawn(self, launcher, cu):
         with open(launch_script_name, "w") as launch_script:
             launch_script.write('#!/bin/bash -l\n\n')
 
-            launch_script.write("echo script start_script `%s` >> %s/PROF\n" % (cu['gtod'], cu_tmpdir))
+            if 'RADICAL_PILOT_PROFILE' in os.environ:
```

Comment:
No, not really working on it.  But I also don't think we do need a  cfg flag right now? (I think this is what you refer to?)